### PR TITLE
Atomizes darkmode pr

### PR DESCRIPTION
**Pull request by Kmc2000**
:cl: Kmc2000
add: Darkmode is now enabled by default. 
/:cl:

Atomization of: https://github.com/DDMers/Deep-Space-13/pull/23

### CODE REVIEW

**Review by alexkar598 — CHANGES_REQUESTED** [review]
Screenshots required for UI change :)

**Review by alexkar598 — CHANGES_REQUESTED** [review]
Post screenshot for UI change :)

**Comment by alexkar598** [comment]
Well fuck,sorry for double post

**Comment by Kmc2000** [comment]
img

**Comment by Kmc2000** [comment]
How to approve my own PR ????

**Comment by Kmc2000** [comment]
img

File: code/modules/goonchat/browserassets/css/browserOutput.css [hunk 1]
```diff
@@ -10,7 +10,7 @@ html, body {
 	color: #000000;
 }
 body {
-	background: #fff;
+	background: #ECE3E3;
```
Comment (alexkar598) [inline code/modules/goonchat/browserassets/css/browserOutput.css:13]: Chat color aint working

File: interface/skin.dmf [hunk 2]
```diff
@@ -65,26 +65,30 @@ window "mainwindow"
...
+		background-color = #272727
 		saved-params = "splitter"
 		left = "mapwindow"
 		right = "infowindow"
 		is-vert = true
 	elem "input"
 		type = INPUT
-		pos = 3,420
-		size = 517x20
+		pos = 5,420
```
Comment (alexkar598) [inline interface/skin.dmf]: Why the position change?

**Review by alexkar598 — DISMISSED** [review]

File: code/modules/goonchat/browserassets/css/browserOutput.css (hunk 1 above)
Comment (Kmc2000) [reply code/modules/goonchat/browserassets/css/browserOutput.css:13]: Intended

Comment (Jalleo) [reply code/modules/goonchat/browserassets/css/browserOutput.css:13]: Its just slightly darker

**Review by Jalleo — APPROVED** [review]
We will want to make this a config pref for people.

**Pull request merged**